packages/tocco-ui-showcase/src/components/PropsTable.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+
+const PropsTable = (props) => {
+  var rows = []
+
+  for (var propName in props.props) {
+    if (props.props.hasOwnProperty(propName)) {
+      var prop = props.props[propName]
+      rows.push(
+        <tr key={propName}>
+          <td>{propName}</td>
+          <td>{prop.description}</td>
+          <td>{prop.required && <span className="glyphicon glyphicon-exclamation-sign required-prop"/> }</td>
+          <td>{prop.type.name}</td>
+        </tr>
+      )
+    }
+  }
+
+  return (
+    <div>
+      <h4>Properties</h4>
```

Comment:
Maybe better to work with `<section>` and `<h1>` or something like that. `<h4>` doesn't make sense here if you take an isolated look at this component (without knowing the markup of the parent components).